jenkins/opensearch/perf-test.jenkinsfile

```diff
@@ -272,7 +272,7 @@ pipeline {
                     "test-single-security-${BUILD_ID}-${ARCHITECTURE}-${BUILD_NUMBER}"
                     ]
                     withCredentials([string(credentialsId: 'jenkins-aws-account-public', variable: 'AWS_ACCOUNT_PUBLIC')]) {
-                        withAWS(role: 'opensearch-test', roleAccount: "${AWS_ACCOUNT_PUBLIC}", duration: 900, roleSessionName: 'jenkins-session', region: 'eu-west-1') {
+                        withAWS(role: 'cfn-set-up', roleAccount: "${AWS_ACCOUNT_PUBLIC}", duration: 900, roleSessionName: 'jenkins-session', region: 'eu-west-1') {
```

Comment:
I am not getting why the role needs to be changed.

I checked and the new policies are already in opensearch-test role, and the cfn-set-up role not even exist.

You added a new policy to an existing role.

Reply:
cfn-set-up is the role actually used to deploy the clusters internally. Lot of hacky ways due to opensearch-infra being private repo. The Jenkins main node and agent node can assume `cfn-set-up` so we are good.

Reply:
Hi,

We need to add these items:
1. Add a new secret to host the proper test cluster account id (named `odfe-aws-account-test` possible?).
2. change the withCredentials section to retrieve the account id from `odfe-aws-account-test`.
3. revert previous change that add the permissions on wrong role.

cc: @jordarlu @gaiksaya 

Thanks.

Reply:
Thanks for the suggestions , @peterzhuamazon @gaiksaya .. I made the update accordingly

Reply:
After our dicussion, we decided the name of the new ID would be   perf-test-account-id